src/components/Button/Button.tsx

```diff
@@ -34,14 +36,16 @@ const Button = (props: ButtonProps) => {
         ...rest
     } = props;
     const defaultImgPosition = 'left';
-
+    const defaultEvent = useAnalyticsEvent(DefaultEventNames.Button, url);
+    const handleAnalytics = useAnalytics(defaultEvent);
```

Comment:
if useAnalyticsEvent and useAnalytics are always using at the same time, may be we can do like this
`const handleAnalytics = useAnalytics({name: DefaultEventNames.Button, url});`? and create event inside of useAnalytics?

Reply:
ok